lib/msal-browser/src/error/BrowserAuthError.ts

```diff
@@ -7,107 +7,16 @@ import { AuthError } from "@azure/msal-common/browser";
-        "Network request failed: If the browser threw a CORS error, check that the redirectUri is registered in the Azure App Portal as type 'SPA'",
-    [BrowserAuthErrorCodes.getRequestFailed]:
-        "Network request failed. Please check the network trace to determine root cause.",
-    [BrowserAuthErrorCodes.failedToParseResponse]:
-        "Failed to parse network response. Check network trace.",
-    [BrowserAuthErrorCodes.unableToLoadToken]: "Error loading token to cache.",
-    [BrowserAuthErrorCodes.cryptoKeyNotFound]:
-        "Cryptographic Key or Keypair not found in browser storage.",
-    [BrowserAuthErrorCodes.authCodeRequired]:
-        "An authorization code must be provided (as the `code` property on the request) to this flow.",
-    [BrowserAuthErrorCodes.authCodeOrNativeAccountIdRequired]:
-        "An authorization code or nativeAccountId must be provided to this flow.",
-    [BrowserAuthErrorCodes.spaCodeAndNativeAccountIdPresent]:
-        "Request cannot contain both spa code and native account id.",
-    [BrowserAuthErrorCodes.databaseUnavailable]:
-        "IndexedDB, which is required for persistent cryptographic key storage, is unavailable. This may be caused by browser privacy features which block persistent storage in third-party contexts.",
-    [BrowserAuthErrorCodes.unableToAcquireTokenFromNativePlatform]: `Unable to acquire token from native platform.  ${ErrorLink}`,
-    [BrowserAuthErrorCodes.nativeHandshakeTimeout]:
-        "Timed out while attempting to establish connection to browser extension",
-    [BrowserAuthErrorCodes.nativeExtensionNotInstalled]:
-        "Native extension is not installed. If you think this is a mistake call the initialize function.",
-    [BrowserAuthErrorCodes.nativeConnectionNotEstablished]: `Connection to native platform has not been established. Please install a compatible browser extension and run initialize().  ${ErrorLink}`,
-    [BrowserAuthErrorCodes.uninitializedPublicClientApplication]: `You must call and await the initialize function before attempting to call any other MSAL API.  ${ErrorLink}`,
-    [BrowserAuthErrorCodes.nativePromptNotSupported]:
-        "The provided prompt is not supported by the native platform. This request should be routed to the web based flow.",
-    [BrowserAuthErrorCodes.invalidBase64String]:
-        "Invalid base64 encoded string.",
-    [BrowserAuthErrorCodes.invalidPopTokenRequest]:
-        "Invalid PoP token request. The request should not have both a popKid value and signPopToken set to true.",
-    [BrowserAuthErrorCodes.failedToBuildHeaders]:
-        "Failed to build request headers object.",
-    [BrowserAuthErrorCodes.failedToParseHeaders]:
-        "Failed to parse response headers",
-    [BrowserAuthErrorCodes.failedToDecryptEarResponse]:
-        "Failed to decrypt ear response",
-};
+export function getDefaultErrorMessage(code: string): string {
+    return `See https://aka.ms/msal.js.errors#${code} for details`;
```

Comment:
Have you tested this? We may not be allowed to use `.` in the short links

Reply:
Yep, this works